modules/simple-payments/paypal-express-checkout.js

```diff
@@ -9,6 +9,7 @@
 /* jshint unused:false */
 var PaypalExpressCheckout = {
 	sandbox: true,
+	$purchaseMessageContainer: null,
```

Comment:
It's jQuery convention. If something is jQuery object, we use `$` in the var name to state that.